### DISCUSSION TITLE
How I can optimize my searchs?

### DISCUSSION THREAD

**Discussion by ryosoftware** [organicmaps/organicmaps#7044] [Q&A]
I love Organic Maps, especially the fact that I don't share my location with Google or others.
I have no problems with the navigation, which generally works quite well, despite not having traffic management (I can live without it), but the worst thing is the address search. When I enter an address many times the street does not appear or streets appear that are not the one I am looking for. For example, when searching for "Ferraz 4 Madrid" the "Madrid street" appears in towns that are relatively close or not to my location but I'm searching "Ferraz" street at city of Madrid. How could I do this to narrow my search to the results I really want? (I don't have that problem with Google Maps, for example).

**Comment by biodranik** [comment]
Please follow https://github.com/organicmaps/organicmaps/issues/7616
